shell/config/product/explorer.js

```diff
@@ -103,6 +103,9 @@ export function init(store) {
     componentForType(WORKLOAD_TYPES[key], WORKLOAD);
   }
 
+  // Deal with POD (it wasn't registered as belonging to the workloads, so it wasn't using the list view of workloads)
+  componentForType(POD, WORKLOAD);
```

Comment:
Pods both are and aren't workloads. Why is this needed?

Importantly though they have their own detail and edit components which are now bypassed with this line. interestingly edit works, but detail is broken.

Reply:
It was to try and fix https://github.com/rancher/dashboard/pull/10126#discussion_r1423204878 . This one was a "bug" that already existed. I prefer to revert this change and have the "same old" bug rather than breaking a complete page.

The problem is that the PODS list view currently uses a default list view and I've just noticed that if you switch between resource list INSIDE the same PRODUCT to another that uses the default list view it doesn't re-adjust the scrolling.

Yesterday I thought I understood why, but apparently I still don't 😅 

I'll revert this change.